html/js/routes/aimetrics.js

```diff
@@ -500,5 +508,25 @@ routes.push({ path: '/aimetrics/:userId?/:sessionId?', name: 'aimetrics', compon
       let percentage = (credits / total) * 100;
       return `${percentage.toFixed(1)}%`;
     },
+    formatDHM(durationMs) {
```

Comment:
Move this to app.js next to the other date formatters. Call it formatLongDuration(). Ideally, it would show only "d" when > 24h, only "h" when > 60m, etc.

Ex: 1.5d
Ex: 2.3h
Ex: 45m
Ex: 1m